zoo/services/templates/services/service_detail.html

```diff
@@ -188,7 +197,36 @@ <h5 class="ui marginless header">
     </div>
   </div>
   {% endif %}
+{% endblock %}
 
+{% block extra_components %}
+{% verbatim %}
+<div class="ui stackable grid tab" data-tab="openapi">
+  {% if object.repository %}
+  {% endverbatim %}
+  <div id="openapi-visualizer" data-url="{% url 'repo_openapi' object.repository.provider object.repository.remote_id %}">
+  {% verbatim %}
```

Comment:
What is intended to be in the `verbatim` block here? It looks like the first verbatim ends on line 206 and the second one starts at line 208.

Reply:
And in fact, it is improperly nested as I can see the following text in the browser:

![Screenshot from 2019-11-14 11-30-57](https://user-images.githubusercontent.com/2439357/68849446-6116bb80-06d2-11ea-87a5-9e13b9aead82.png)

Reply:
It's just adding a `non-verbatim` space inside the `verbatim` block from 203 to 229. Because I need to render the URL. 

The verbatim block is needed because Vue.js has the same templating syntax as Django, and Django would try to resolve it otherwise.

Not a huge fan of the solution, it could be done with fancier approaches, but I don't think it's worth the effort. We'll end up switching to an SPA eventually.

Reply:
Oops... don't know what I was thinking of with that solution... 😅 it was really bad. Fixed now